tests/Drupal/bad/bad.php

```diff
@@ -838,6 +838,26 @@ function test28() {
 enum aBad_NameForAnEnum {
 }
 
+/**
+ * Test PHP attributes.
+ */
+class TestPhpAttributes {
```

Comment:
please move this test to ScopeIndentUnitTest.inc and adapt the fixed file as well as the list of errors in ScopeIndentUnitTest.php.

Reply:
I think I addressed both of these!